### PR TITLE
Spelling, Grammar and md lint

### PR DESCRIPTION
**Pull request by Dax911**
## Changes

- What does this change?
Just fixes small spelling, grammar and markdown across the readme's 
- Before/after screenshots can help as well: 
its spelling, grammar and spacing, I really don't think I need to.
- Don't forget a changeset! `pnpm exec changeset`:
not needed

## Testing

There were no actual changes to code only the readme's

## Docs

The readmes were the things updated. 

I pulled this down to make some changes of my own, but before I could start my linter and grammarly plugins were yelling at me. It annoyed me enough that I fixed the grammar and basic markdown so they would stop.  Little things like making sure code blocks had a language declared etc.

### CODE REVIEW

File: examples/docs/README.md
Comment (bluwy) [inline examples/docs/README.md:1]: I don't think this file is correctly changed. The grammar was fine, and it removes necessary code and whitespace.

**Comment by sarah11918** [comment]
Thanks, @Dax911 for some of these helpful fixes!  :raised_hands: You've caught some great stuff here, but @bluwy is right that I can see not every change is necessarily something we'd do here in our docs, so I'll have to evaluate everything one at a time to see what you've changed.

This is more a heads up that a PR like this is a *lot* for one PR, because it mixes a bunch of little things. So, it'll take some time to go through and scan everything pretty closely, but we are on it!

In future, it would have been totally fine to submit *one* PR that adds the `bash` to all the code samples, one PR that changes tabs/spaces, etc. That allows us to focus one one thing at a time in so many files and makes those super easy to merge quickly. With a PR like this, it just takes a bit more slogging to get through it all. And, if there's a change that isn't something we want (like removing some intentional whitespace, for example), it holds up getting the other stuff merged. And it's a lot of scrolling through various detail-oriented changes to check on everything.

**Comment by Dax911** [comment]
Yep I thought it might be large so there should be two separate commits one that deals w the grammar spelling etc and the other is the markdown. So u can go ahead and split it up if you want. Wanted to make sure u had options. I will def keep it smaller going forward.

**Comment by Dax911** [comment]
Totally I'd be glad to double check too and see where I can improve/find out what's more in line w how u do things.

**Comment by Jutanium** [comment]
Hey, thanks for this!

There's a lot going on in this PR. Some of which is helpful - changing "based off of" to "based on" - and some changes that either don't change the final output or that we can't accept.

This kind of PR has proven too difficult to go through given the number of files we'd need to review manually and given the difficulty to accept only some of these changes.

I see Sarah has given some advice for submitting smaller, focused PRs. Those are much easier to review and provide a better way to figure out which changes we want to accept.

Unfortunately, we have to close this PR, but please feel free to take Sarah's advice above and submit multiple PRs!

**Pull request closed without merging**